usecase/src/main/java/org/mobilitydata/gtfsvalidator/usecase/ValidateHeadersForFile.java

```diff
@@ -44,6 +54,12 @@ public ValidateHeadersForFile(final GtfsSpecRepository specRepo,
         this.resultRepo = resultRepo;
     }
 
+    /**
+     * Use case execution method: for a file, checks the presence of all headers marked as "required" in the
+     * GTFS specification. A {@code MissingHeaderNotice} is generated each time a required header is missing.
```

Comment:
I would `{@link` the classes in this Javadoc so you can click on them within the IDE.